tfx/benchmarks/benchmark_base.py

```diff
@@ -27,14 +28,77 @@
     "Beam runner to use - any runner name accepted by "
     "apache_beam.runners.create_runner")
 
+DEFAULT_MODE = "default"
+LOCAL_SCALED_EXECUTION_MODE = "local_scaled_execution"
+CLOUD_DATAFLOW_MODE = "cloud_dataflow"
+FLINK_ON_K8S_MODE = "flink_on_k8s"
+modes = [DEFAULT_MODE, LOCAL_SCALED_EXECUTION_MODE, CLOUD_DATAFLOW_MODE,
+         FLINK_ON_K8S_MODE]
+
+beam_pipeline_mode = DEFAULT_MODE
+num_workers_str = "1"
```

Comment:
nit: make this an int, then use `"--num_workers=%d" % num_workers` to interpolate

Reply:
Done, made it an int.